usort/stdlibs.py

```diff
@@ -5,6 +5,6 @@
 
 from typing import FrozenSet
 
-from stdlibs import py27, py3
+from stdlibs import py3
 
-STDLIB_TOP_LEVEL_NAMES: FrozenSet[str] = py27.module_names | py3.module_names
+STDLIB_TOP_LEVEL_NAMES: FrozenSet[str] = py3.module_names
```

Comment:
Previous code created a new object. I don't think we need to since this is a `frozenset`, but LMK if you want me to throw a `.copy()` in there to be absolutely sure.

Reply:
I'm fine as-is.